Sources/PackageModel/Package.swift

```diff
@@ -91,6 +91,20 @@ public final class Package {
         case noOrigin(String)
     }
 }
+extension Package.Error: CustomStringConvertible {
+   public var description: String {
+        switch self {
+        case .noManifest(let baseURL, let version):
+            var string = "\(baseURL) has no manifest"
+            if let version = version {
+                string += " for version \(version)"
+            }
+            return string
+        case .noOrigin(let string):
```

Comment:
I don't know if we're even using this error anymore, can you double check?

Reply:
yeah it was not anywhere i removed this case